scene/resources/3d/importer_mesh.h

```diff
@@ -82,6 +82,52 @@ class ImporterMesh : public Resource {
 
 	Size2i lightmap_size_hint;
 
+	struct MergedAttribute {
+		Vector3 normal;
+		Vector<int> primary_bone_influence;
+	};
```

Comment:
This should not have a `Vector<int>` for every vertex. it is inefficient, especially given that there is at most a small constant number of influences per vertex

I think it should be possible with one influence and one weight, maybe two influences at most but I'd like to see a counter-example.
Remember that mesh simplify will try to avoid merging vertices with different values here:

Even if the vertex contains a lot more influences, the one or two most dominant influences should be enough to ensure the vertices are considered different.

Reply:
The linked issue ‘s glb fails to decimate correctly if we limit to one or two influences.

Reply:
[test.glb.zip](https://github.com/godotengine/godot/files/14873650/test.glb.zip)

# Limited to 1 influence

https://github.com/godotengine/godot/assets/32321/51234fdb-7bc1-41d5-b4e5-04a47a357e48

# Limited to 8 influences

https://github.com/godotengine/godot/assets/32321/10dc77f3-3d3d-4ed7-8a09-e2cb13bed3b1